src/Sentry/Protocol/IScope.cs

```diff
@@ -129,29 +85,22 @@ public User User
         /// </remarks>
         /// <example> { "fingerprint": ["myrpc", "POST", "/foo.bar"] } </example>
         /// <example> { "fingerprint": ["{{ default }}", "http://example.com/my.url"] } </example>
-        public IEnumerable<string> Fingerprint => InternalFingerprint ?? Enumerable.Empty<string>();
+        IEnumerable<string> Fingerprint { get; set; }
 
         /// <summary>
         /// A trail of events which happened prior to an issue.
         /// </summary>
         /// <seealso href="https://docs.sentry.io/platforms/dotnet/enriching-events/breadcrumbs/"/>
-        public IEnumerable<Breadcrumb> Breadcrumbs => InternalBreadcrumbs ??= new ConcurrentQueue<Breadcrumb>();
+        ICollection<Breadcrumb> Breadcrumbs { get; }
 
         /// <summary>
         /// An arbitrary mapping of additional metadata to store with the event.
         /// </summary>
-        public IReadOnlyDictionary<string, object?> Extra
-            => InternalExtra ??= new ConcurrentDictionary<string, object?>();
+        IDictionary<string, object?> Extra { get; }
```

Comment:
Same here, not expecting to let get a hold of the mutable reference (of course they could type cast but that's beside the point).

The Scope in other SDKs is not even readable. This was made readable but not mutable (except through the APIs we defined like `SetExtra`